lib/support/live_select_web/live/showcase_live.html.heex

```diff
@@ -326,7 +345,10 @@
             <.copy_to_clipboard_icon />
           </button>
         </div>
-        <Render.live_select opts={Settings.live_select_opts(@settings_form.data, true)} />
+        <Render.live_select
+          custom_option_html={@settings_form[:custom_option_html].value}
```

Comment:
the argument of the function component should also be renamed to `options_styled_as_checkboxes`